src/QueryPipeline/RemoteInserter.cpp

```diff
@@ -34,6 +34,8 @@ RemoteInserter::RemoteInserter(
             = CurrentThread::get().thread_trace_context;
     }
 
+    server_revision = connection.getServerRevision(timeouts);
```

Comment:
Move to [member initializer list](https://github.com/ClickHouse/ClickHouse/pull/34132/files#diff-57e439597c788d196efd2024e3e1a99113e82b6b4ecf9f03fbf09b43a759d03dL27)